### DISCUSSION TITLE
Shelter types - How to assign own info text?

### DISCUSSION THREAD

**Discussion by ERYpTION** [openstreetmap/iD#10436] [Q&A]
When setting a 'amenity=shelter' the info text ("A small structure for protection against bad weather conditions") and documentation-link is the same for some of the shelter types (for example gazebo, lean_to and basic_hut) and while others have their own info text and link (for example picnic_shelter and public_transport). 

What determines whether or not a shelter amenity gets its own info and description? I would especially like the lean_to to have its own info and wiki-link.

**Comment by tordans** [comment]
Quick answer from remote: please check the "id tagging schema" repo and have a look at the info on wiki data items which handle the info-i text and the `reference` key to specify which wiki link to use. The recently merged PR for community center might also be a good read on how to add docs to sub presets. 

Can write more later if needed.

**Reply by ERYpTION** [reply]
Thank you for the quick response! However, I have to admit this is a bit over my head. I took a look but wasn't able to find the relevant information in the repo (that I understand). So I probably am not able to handle this myself yet. Hopefully, someone with more experience will eventually pick it up.

**Reply by tordans** [reply]
Will send you better input later.

**Reply by ERYpTION** [reply]
I am still interested in learning how to do this if anyone can point me in the right direction. Thanks.

**Reply by tordans** [reply]
@ERYpTION please have a look at https://github.com/openstreetmap/id-tagging-schema/pull/1257/files which goes into details on the osm wikidata item an the "reference" key and links to further info.

Sorry for the delay, thanks for the ping.

**Reply by ERYpTION** [reply]
Thank you for the follow up. Unfortunately it confirms that I am out of my depth here.

**Reply by tordans** [reply]
This is the data item for this tag https://wiki.openstreetmap.org/wiki/Item:Q5007 which can be edited similar to a wiki page.

**Reply by ERYpTION** [reply]
But as far as I can see this data item only concerns the main amenity=shelter not shelter type like lean_to?
